services/brig/test/integration/API/User/Client.hs

```diff
@@ -481,6 +490,10 @@ testRemoveClient hasPwd brig cannon = do
 
 -- The testRemoveClientShortPwd test conforms to the following testing standards:
 -- @SF.Provisioning @TSFI.RESTfulAPI @S2
+--
+-- The test checks if a client can be deleted by providing a too short password.
+-- This is done by using a single-character password, whereas the minimum is 6
+-- characters. The client deletion attempt fails as expected.
```

Comment:
your description is correct, but what a weird test to have!  i suggest we add a test with a syntactically valid but wrong password, and make sure that password verification also takes place?  (not strictly in the scope of this PR, but it's easy to add now, it makes sense, and it will potentially save us the trouble of doing another review round later.)

Reply:
This sounds like a good idea! I'll add the test you proposed as part of this PR.